web/viewer.css

```diff
@@ -1264,6 +1223,49 @@ dialog :link {
 }
 
 #secondaryToolbar {
+  .toolbarButton {
+    border-radius: 0;
+    display: inline-block;
+    height: auto;
+    margin: 0;
+    padding: 0 0 1px;
+    padding-inline-start: 36px;
+    position: relative;
+    min-height: 26px;
+    min-width: 100%;
+    text-align: start;
+    white-space: normal;
+    width: auto;
+
+    &::before {
+      inset-inline-start: 12px;
+      opacity: var(--doorhanger-icon-opacity);
+      top: 5px;
+    }
+
+    &:is(:hover, :focus-visible) {
+      background-color: var(--doorhanger-hover-bg-color);
+      color: var(--doorhanger-hover-color);
+    }
+
+    & > span {
```

Comment:
Is the `&` actually necessary, since I *believe* that it's implicit if written as follows?
```css
> span {
```

Reply:
You're right, it's not. I have removed it in the new commit.